lib/SPIRV/OCLUtil.cpp

```diff
@@ -1358,6 +1358,8 @@ bool isSamplerTy(Type *Ty) {
   if (auto *TPT = dyn_cast_or_null<TypedPointerType>(Ty)) {
     auto *STy = dyn_cast_or_null<StructType>(TPT->getElementType());
     return STy && STy->hasName() && STy->getName() == kSPR2TypeName::Sampler;
+  } else if (auto *TET = dyn_cast_or_null<TargetExtType>(Ty)) {
```

Comment:
If believe, that's what clang-tidy is asking for:
```suggestion
  }
   if (auto *TET = dyn_cast_or_null<TargetExtType>(Ty)) {
```